### PR TITLE
updated chaindatafetcher block group message

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
## Proposed changes

- the block group output is udpated to the api GetBlockWithConsensusInfo
- new fields round and originProposer is added
- moved the logic of retrieving consensus info to consensus engine
  instead of api package

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues
- https://github.com/klaytn/klaytn/issues/1531

### CODE REVIEW

**Review by ethan-kr — APPROVED** [review]

File: consensus/clique/clique.go [hunk 1]
```diff
@@ -442,6 +442,11 @@ func (c *Clique) snapshot(chain consensus.ChainReader, number uint64, hash commo
 	return snap, err
 }
 
+// GetConsensusInfo is not used for Cliqueue engine
```
Comment (kjhman21) [inline consensus/clique/clique.go:445]:
```suggestion
// GetConsensusInfo is not used for Clique engine
```

File: consensus/consensus.go [hunk 2]
```diff
@@ -111,6 +111,9 @@ type Engine interface {
 
 	// CreateSnapshot does not return a snapshot but creates a new snapshot at a given point in time.
 	CreateSnapshot(chain ChainReader, number uint64, hash common.Hash, parents []*types.Header) error
+
+	// GetConsensusInfo returns consensus information regarind to the given block number.
```
Comment (kjhman21) [inline consensus/consensus.go:115]:
```suggestion
	// GetConsensusInfo returns consensus information regarding to the given block number.
```

File: consensus/consensus.go [hunk 3]
```diff
@@ -149,3 +152,10 @@ type Istanbul interface {
 	// SetChain sets chain of the Istanbul backend
 	SetChain(chain ChainReader)
 }
+
+type ConsensusInfo struct {
```
Comment (kjhman21) [inline consensus/consensus.go:156]: Do we need to redefine this structure?

Comment (kjhman21) [reply consensus/consensus.go:156]: ah, it is moved from `consensus/istanbul/backend/api.go`? Is that right?

Comment (jeongkyun-oh) [reply consensus/consensus.go:156]: Yes, right.

File: consensus/clique/clique.go (hunk 1 above)
Comment (jeongkyun-oh) [reply consensus/clique/clique.go]: Updated. Thanks.

File: consensus/consensus.go (hunk 2 above)
Comment (jeongkyun-oh) [reply consensus/consensus.go]: Updated. Thanks.

**Review by kjhman21 — APPROVED** [review]
LGTM.

**Review by aidan-kwon — APPROVED** [review]

**Review by mckim19 — APPROVED** [review]

**Comment by jeongkyun-oh** [comment]
Test for this PR

- checked newly added/original fields with Kafdrop.

스크린샷 2022-12-01 오후 4 55 20

**Pull request merged**